src/core/MenuItem/style.ts

```diff
@@ -1,19 +1,53 @@
 import styled from "@emotion/styled";
-import { MenuItem } from "@material-ui/core";
-import { Black } from "../styles/common/constants/black";
-import { FontWeights } from "../styles/common/constants/fontWeights";
-import { Gray } from "../styles/common/constants/gray";
+import { Checkbox, MenuItem } from "@material-ui/core";
+import { Check } from "@material-ui/icons";
 import { fontBody } from "../styles/common/mixins/fonts";
-import { getPalette, getSpacings } from "../styles/common/selectors/theme";
+import {
+  getColors,
+  getFontWeights,
+  getIconSizes,
+  getPalette,
+  getSpacings,
+} from "../styles/common/selectors/theme";
 
 const fontBodyXs = fontBody("xs");
 
 export const StyledMenuItem = styled(MenuItem)`
   ${(props) => {
+    const { selected } = props;
+    const colors = getColors(props);
+    const fontWeights = getFontWeights(props);
     const spacings = getSpacings(props);
 
+    const primary = colors?.primary[400];
+
     return `
       padding: ${spacings?.s}px ${spacings?.m}px;
+
+      &.MuiListItem-root.Mui-selected,
+      &.MuiListItem-root.Mui-selected:hover {
+        background-color: transparent;
+
+        &:hover {
+          background-color: ${colors?.gray[100]};
```

Comment:
When I inspect the hover background color in my browser, it's showing rgba(0, 0, 0, 0.04)/#000000a, however it should be #F8F8F8. Can you make sure the color style props are aligned to the right hex codes from our [SDS color page](https://www.figma.com/file/7mEbMOSLShNxOSV2pyl0cU/Science-Design-System?node-id=2120%3A268)?